database/migrations/2021_05_22_092019_add_any_or_all_flag_to_waiting_list_flags.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use App\Models\Training\WaitingList;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddAnyOrAllFlagToWaitingListFlags extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('training_waiting_list', function (Blueprint $table) {
+            $table->string('flags_check')->default(WaitingList::ALL_FLAGS);
```

Comment:
For table neatness?

```suggestion
            $table->string('flags_check')->default(WaitingList::ALL_FLAGS)->after('department');
```

Reply:
Good call https://github.com/VATSIM-UK/core/pull/2184/commits/fbc9863153adea260a53833b01d379479c122eed